src/enviable/cli.clj

```diff
@@ -24,11 +24,19 @@
 (defn name-cell [{::core/keys [name]}]
   (table/cell name))
 
-(defn input-cell [{::core/keys [input]}]
-  (table/cell input))
+(def sensitive-value-cell
+  (table/cell "(sensitive value hidden)"))
```

Comment:
Could we just do `(table/cell "******")` instead?

Reply:
Yeah I actually started off with that, but I wasn't sure if it would be less obvious from the output what was happening... maybe starred out password are common enough that it's not an issue though. Happy to switch it over.

Reply:
👍